agent.go

```diff
@@ -26,6 +26,19 @@ import (
 // AgentType describes the type of guest agent a Pod should run.
 type AgentType string
 
+// ProcessListOptions contains the options used to list running
+// processes inside the container
+type ProcessListOptions struct {
+	// Format describes the output format to list the running processes
+	Format string
+
+	// Args contains the list of arguments to run ps command
```

Comment:
Again, the agent PR shows that if this isn't set, it default to sane values so I wonder if it would be worth adding "(optional)" at the end of the comment here?

Reply:
default value is set in the runtime https://github.com/clearcontainers/runtime/pull/444/files#diff-b985e723bb136b291af1a569f76616d4R67